app/controllers/v1/facilities/va_ccp_controller.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'will_paginate/array'
+class V1::Facilities::VaCcpController < FacilitiesController
+  def urgent_care
+    ppms_api_results = ppms_api.pos_locator(ppms_params).collect do |result|
+      PPMS::Provider.new(
+        result.attributes.transform_keys { |k| k.to_s.snakecase.to_sym }
+      )
+    end.uniq(&:id).paginate(pagination_params)
+
+    lighthouse_api_results = lighthouse_api.get_facilities(
+      lighthouse_params.merge(
+        type: :health,
+        services: ['UrgentCare']
+      )
+    )
+
+    providers_facilities = PPMS::ProviderFacility.new
+    providers_facilities.providers = ppms_api_results
+    providers_facilities.facilities = lighthouse_api_results
```

Comment:
Could we refactor lines 6-21 into a command or class? The code is very clean & readable (thank you! 🙇), but the merging of results seems like a `PPMS::ProviderFacility` concern, more than a controller concern.

Reply:
Yes but No. PPMS::ProviderFacility is more of a dummy object that makes fast_jsonapi 'include' all of the results from each resource. This is an odd limitation I ran into, the response will only use one Serializer and I could not find another way to get both objects into the response. I will move the logic into the ProviderFacility. it will make it a bit different from how the other facility controllers handle it.